source/linux/How_to_Guides/Target/How_to_BeagleBadge_getting_started.rst

```diff
@@ -0,0 +1,151 @@
+###########################
+BeagleBadge Getting Started
+###########################
+
+Overview
+========
+
+The BeagleBadge is a compact development platform from BeagleBoard.org powered by the TI AM62L SoC.
+Designed for portable and low-power applications, it features built-in Wi-Fi and Bluetooth, multiple
+low power modes, and an integrated fuel gauge for precise battery power monitoring.
+
+The board provides a rich interface including an e-paper connector, DSI connector, Grove expansion,
+seven-segment displays, and an RGB LED. Fully supported in TI releases, the BeagleBadge offers flexible
+boot options (OSPI, UART, SD, USB-DFU) and runs Zephyr OS or Linux via Debian, Arago, or Buildroot,
+making it an ideal open-source solution for modern IoT and HMI projects.
+
+Supported OS
+============
+
+- Linux : https://github.com/TexasInstruments/ti-linux-kernel branch: ti-linux-6.12.y
+- Zephyr : https://github.com/glneo/zephyr branch: openamp-am62x-sk-m4
+
```

Comment:
We currently don't list supported operating systems anywhere in the docs. This is a byproduct of every build being operating system specific.